Symfony/CS/Fixer/Symfony/MethodSeparationFixer.php

```diff
@@ -156,6 +159,42 @@ private function fixSpaceAboveMethod(Tokens $tokens, $classStart, $methodIndex)
             }
         }
 
+        // deal with comments above a method
+        if ($tokens[$nonWhiteAbove]->isGivenKind(T_COMMENT)) {
+            if ($firstMethodAttrIndex - $nonWhiteAbove === 1) {
+                // no white space found between comment and method start
+                $this->correctLineBreaks($tokens, $nonWhiteAbove, $firstMethodAttrIndex, 1);
+
+                return;
```

Comment:
I would keep here the return.
@GrahamCampbell as you said few days ago - it's a matter of code style, and if we you almost everywhere early-return pattern, then your pointing to change them to if-elseif in every new PR is pointless. Both are good, but be consistence with codebase.